examples/basic/topology_optimization_cantilever_beam.py

```diff
@@ -0,0 +1,188 @@
+from Ansys.Mechanical.DataModel.MechanicalEnums import *
+
+
+def display_image(image_name):
+    path = os.path.join(os.path.join(cwd, image_name))
+    image = mpimg.imread(path)
+    plt.figure(figsize=(15, 15))
+    plt.axis("off")
+    plt.imshow(image)
+    plt.show()
+
+
+cwd = os.path.join(os.getcwd(), "out")
+
+structural_mechdat_file = download_file(
+    "cantilever.mechdat", "pymechanical", "embedding"
+)
+app.open(structural_mechdat_file)
+STRUCT = ExtAPI.DataModel.Project.Model.Analyses[0]
+assert str(STRUCT.ObjectState) == "Solved"
+STRUCT_SLN = STRUCT.Solution
+STRUCT_SLN.Solve(True)
+assert str(STRUCT_SLN.Status) == "Done", "Solution status is not 'Done'"
+
+##############################################
+# Configure graphics for image export
+
+ExtAPI.Graphics.Camera.SetSpecificViewOrientation(
+    Ansys.Mechanical.DataModel.Enums.ViewOrientationType.Iso
+)
+image_export_format = Ansys.Mechanical.DataModel.Enums.GraphicsImageExportFormat.PNG
+settings_720p = Ansys.Mechanical.Graphics.GraphicsImageExportSettings()
+settings_720p.Resolution = (
+    Ansys.Mechanical.DataModel.Enums.GraphicsResolutionType.EnhancedResolution
+)
+settings_720p.Background = Ansys.Mechanical.DataModel.Enums.GraphicsBackgroundType.White
+settings_720p.Width = 1280
+settings_720p.Capture = Ansys.Mechanical.DataModel.Enums.GraphicsCaptureType.ImageOnly
+settings_720p.Height = 720
+settings_720p.CurrentGraphicsDisplay = False
+
+###############################
+# Structural Analsys Results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+###############################
+# Total Deformation
+
+STRUCT_SLN.Children[1].Activate()
+ExtAPI.Graphics.Camera.SetFit()
+ExtAPI.Graphics.ExportImage(
+    os.path.join(cwd, "total_deformation.png"), image_export_format, settings_720p
+)
+display_image("total_deformation.png")
+
+###############################
+# Equivalent Stress
+
+STRUCT_SLN.Children[2].Activate()
+ExtAPI.Graphics.Camera.SetFit()
+ExtAPI.Graphics.ExportImage(
+    os.path.join(cwd, "equivalent_stress.png"), image_export_format, settings_720p
+)
+display_image("equivalent_stress.png")
+
+##############################################################
+# Topology Optimization
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Set MKS Unit System
+ExtAPI.Application.ActiveUnitSystem = MechanicalUnitSystem.StandardMKS
+
+# Store all main tree nodes as variables
+GEOM = ExtAPI.DataModel.Project.Model.Geometry
+MSH = ExtAPI.DataModel.Project.Model.Mesh
+NS_GRP = ExtAPI.DataModel.Project.Model.NamedSelections
+CONN_GRP = ExtAPI.DataModel.Project.Model.Connections
+MY_TOTAL_VOL = GEOM.Volume.Value
+MY_TOTAL_MA = GEOM.Mass.Value
+
+# Get Structural Analysis and link to Topology optimization
+TOPO_OPT = ExtAPI.DataModel.Project.Model.AddTopologyOptimizationAnalysis()
+TOPO_OPT.TransferDataFrom(STRUCT)
+assert str(TOPO_OPT.ObjectState) == "NotSolved"
+
+# Set None for Optimization Region Boundary Condition Exclusion Region
+# Optimization Region
```

Comment:
```suggestion
# Set ``None`` for optimization region boundary condition exclusion region
# Optimization region
```
This phrase (optimization region boundary condition exclusion region) has five adjectives! It's hard to process. Can it be rewritten as exclusion region for the optimization region's boundary condition?